sigle/src/modules/publicHome/components/PublicHome.tsx

```diff
@@ -358,6 +372,16 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
                 />
               ))}
             </TabsContent>
+            <TabsContent value="following">
```

Comment:
regarding the approach for the tabs I think that taking another approach would be better. The content should be rendered and data fetched only when you select the tab. At the moment we fetch both the following / followers data even tho you never watch this tab. Also the URI should be a different one, that way links can redirect you to the follower tab directly etc. Wdyt ?

Reply:
ok yeah that makes sense. Regarding the URI, would [shallow routing](https://nextjs.org/docs/routing/shallow-routing) work for this?

Reply:
Not sure shallow routing would be the best as you would still have to fetch all the data using this approach no?